### DISCUSSION TITLE
Pixel 7 Pro / NordVPN App

### DISCUSSION THREAD

**Discussion by cgi2099** [Mygod/VPNHotspot#427] [Q&A]
Trying on a Pixel Pro 7, rooted, magisk, zygisk, lsposed

I cannot get this to work with the NordVPN app.

I use the NordVPN app to create the VPN tunnel on the phone and then try to use the VPN Hotspot app. When I connect to the phones hotspot it still shows At&t's IP.

**Comment by Mygod** [comment]
Turn off tethering hardware acceleration.

**Reply by cgi2099** [reply]
It still doesn't work after disabling tethering hardware acceleration.

**Reply by cgi2099** [reply]
> Turn off tethering hardware acceleration did not work.
Do you have another recommendation for this? Turning hardware acceleration off did not work for me.
